scripts/src/chartprreview/chartprreview.py

```diff
@@ -46,7 +46,7 @@ def verify_user(directory, username, category, organization, chart):
 
     data = open(owners_path).read()
     out = yaml.load(data, Loader=Loader)
-    if username not in [x['githubUsername'] for x in out['users']]:
+    if username.strip() not in [x['githubUsername'] for x in out['users']]:
```

Comment:
I don't think we need this, right?

Reply:
yep closing, for reference, users should create pull request against main, not their test branch. The original issue was because in the target branch, the username is not in the OWNERS file.